src/Recordings.cpp

```diff
@@ -568,7 +568,12 @@ PVR_ERROR Recordings::SetRecordingLastPlayedPosition(const kodi::addon::PVRRecor
     time_t timerUpdate = m_timers.m_lastTimerUpdateTime;
     if (lastplayedposition == -1)
     {
-      lastplayedposition = recording.GetDuration();
+      if (recording.GetRecordingTime() + recording.GetDuration() > time(nullptr)) {
```

Comment:
You appear to have changed the `{}` placement for the condition so it looks strange next to the other code.